source/extensions/filters/common/ext_authz/ext_authz_http_impl.cc

```diff
@@ -241,6 +241,7 @@ void RawHttpClientImpl::check(RequestCallbacks& callbacks,
   } else {
     // Do not enforce a sampling decision on this span; instead keep the parent's sampling status.
     auto options = Http::AsyncClient::RequestOptions()
+                       .setSendXff(false)
```

Comment:
Please add a runtime guard for this change. See #26417 for an example if you're not familiar with how to do it.